_drafts/embedded-world-recap.md

```diff
@@ -0,0 +1,155 @@
+welcomed hundreds of people at our booth, with 4 demo stations in active
+use throughout the 3 days of the show. The Memfault socks we brought as
+booth swag were a big hit.
+
+<!-- excerpt start -->
+
+In this post, I will share the technologies I saw at Embedded World 2024 that I
+was most impressed by. I was lucky to have plenty of time to walk from booth to
+booth, and had some fantastic conversation that I hope you will find
+interesting.
+
+<!-- excerpt end -->
+
+![]({% img_url embedded-world-2024/memfault-booth.jpg %})
+
+
+{% include newsletter.html %}
+
+{% include toc.html %}
+
+## Rust, front & center
+
+![]({% img_url embedded-world-2024/hightec.jpg %})
+
+This year, Rust seems to have broken through in the embedded community. Nowhere
+was it more evident than at the Hightec booth. For those of you who are not
+familiar with hightec, they sell qualified builds of open source compilers
+(e.g. GCC) for safety critical industries such as automotive. This year, they
+had a HUGE crab-inspired mascot in the background of their booth to announce
+the availability of their ASIL D qualified Rust compiler. I’m excited to see
+serious ecosystem players like Hightec make such a huge bet on Rust as I
+believe it is a better language and runtime for the embedded world. As an
+aside, we’ve been building some of our SDKs in Rust at Memfault. 
+
+## RISC-V continues to gain popularity
+
+![]({% img_url embedded-world-2024/risc-v.jpg %})
+
+RISC-V had a huge booth hosting multiple ecosystem players. SiFive was there of
+course, but I found it notable to see industrial heavyweights such as Siemens
+exhibiting at the RISC-V booth. There is a lot of energy in the RISC-V
+ecosystem, and we’re starting to see some of our customers use RISC-V chips in
+their products. While I continue to be a big fan of the ARM family of
+architecture it’s great to see alternatives develop, especially open source
+ones.
+
+## AI is on every lips
+
+![]({% img_url embedded-world-2024/tinyml.jpg %})
+
+Perhaps the most popular topic of conversation at Embedded World this year was
+- unsurprisingly - AI at the Edge. The TinyML foundation (pictured above) is
+leading the charge, bringing together exciting startups such as Edge Impulse
+and Imagimob that make it easy to deploy models to your devices. But everyone
+else was talking about AI too! Semiconductors are releasing new development
+kits geared towards AI such as the Infineon PSoC 6 AI EK, software vendors are
+integrating AI features, and research labs are showing off cool new use cases
+for AI such as power efficient Atrial Fibrillation detectors for healthcare use
+cases.
+
+
+## Zephyr is winning hearts & minds
+
+![]({% img_url embedded-world-2024/zephyr.jpg %})
+
+Every year, the Zephyr booth is bigger, showing off more sponsors, and busier.
+2024 was no exception. I’m thrilled to see the energy in behind this open
+source project with an ecosystem of chips, tools, and libraries coming together
+around it. 
+
```

Comment:
We could add a note that emphasizes our connection to Zephyr. Something along the lines of: "As a member of the Zephyr project, Memfault got to sit at the Zephyr booth each day to chat with attendees. We spoke with various embedded developers and semiconductor companies excited about transitioning to Zephyr, or in the case of semiconductor companies, adding support for Zephyr in their SDKs." I got a bunch of folks asking what the upsides of Zephyr were and why they should use it / support it.

A fun add here could be that we hope Zephyr brings more kites next year. I had like 4 different people asking me desperately if I had an extra kite!